schemas/useCreatePasswordSchema.ts

```diff
@@ -0,0 +1,21 @@
+import * as Yup from 'yup';
+import { useTranslation } from 'next-i18next';
+
+const useCreatePasswordSchema = () => {
+  const { t } = useTranslation();
+
+  const schema = Yup.object({
+    password: Yup.string()
```

Comment:
so this has been bothering me for a while, but idk why i didn't comment, why are u doing this? this aren't schemas they are getting to point of custom hooks, why not just put for example passwordRequired as response and then pass it to useTranslation when u render error? i think that would be better than making custom hooks for every scheme like this